source/_posts/zio-errors.md

```diff
@@ -0,0 +1,169 @@
+---
+title: io-errors-handle
+date: 2019-09-06 21:05:38
+tags:
+- IO
+- Side Effects
+- Error Handling
+- Pure Functional Programming
+---
+
+# Error Handling with ZIO in Pure Functional Applications
```

Comment:
Here you mention ZIO, assuming that the reader knows what it is. Later you introduce it. I would go with either way, introduce it right away or assume it known.